gcp/api/server.py

```diff
@@ -797,11 +798,34 @@ def _query_by_generic_version(
   if results:
     return results, cursor
 
+  temp_version = version
   # Try again after normalizing.
   version = osv.normalize_tag(version)
   query = base_query.filter(osv.Bug.affected_fuzzy == version)
```

Comment:
This query then loop is now repeated 3 different times, let's refactor it into it's own function and just call it 3 times. (Specifically 804 to 820)